gef.py

```diff
@@ -7402,81 +7215,81 @@ class GlibcHeapUnsortedBinsCommand(GenericCommand):
     See: https://github.com/sploitfun/lsploits/blob/master/glibc/malloc/malloc.c#L1689."""
 
     _cmdline_ = "heap bins unsorted"
-    _syntax_  = "{:s} [ARENA_ADDRESS]".format(_cmdline_)
+    _syntax_  = f"{_cmdline_} [ARENA_ADDRESS]"
 
-    def __init__(self):
+    def __init__(self) -> None:
         super().__init__(complete=gdb.COMPLETE_LOCATION)
         return
 
     @only_if_gdb_running
-    def do_invoke(self, argv):
+    def do_invoke(self, argv: List) -> None:
         if get_glibc_arena() is None:
             err("Invalid Glibc arena")
             return
 
-        arena_addr = "*{:s}".format(argv[0]) if len(argv) == 1 else __gef_current_arena__
-        gef_print(titlify("Unsorted Bin for arena '{:s}'".format(arena_addr)))
+        arena_addr = f"*{argv[0]}" if len(argv) == 1 else gef.heap.selected_arena
```

Comment:
Something got mixed up here! 

`gef.heap.selected_arena` returns the `GlibcArena` and not a string. This leads to the `heap bins` command not working properly for `unsorted`, `small` and `large`

Reply:
this should be done differently, e.g.:
1) like above in `GlibcHeapFastbinsYCommand` where first `arena` (not `arena_addr`) is defined and then converted to `int` before printing (AND it needs to be checked for `None` before printing)
OR
2) `gef.heap.selected_arena` needs to be explicitly converted to a correct `str` ( so e.g. `hex(int(gef.heap.selected_arena))`)

Reply:
Disregarding those items, out of scope of the PR.

See #785